test/build/import-proxy/package.json

```diff
@@ -0,0 +1,13 @@
+{
+  "private": true,
+  "version": "1.0.0",
+  "name": "@snowpack/test-import-proxy",
+  "description": "A test to make sure that the plugin proxy() hook is working as expected",
+  "scripts": {
+    "start": "snowpack dev",
+    "testbuild": "snowpack build"
+  },
+  "devDependencies": {
+    "snowpack": "*"
+  }
+}
```

Comment:
not a blocker for this PR, but just curious: should we add a CSS modules build test? That’d also be a great proxy test.

Reply:
+1, I'll merge this and then follow up in a separate PR